server-spi/src/main/java/com/tc/spi/Guardian.java

```diff
@@ -20,25 +20,26 @@
 
 import java.util.Properties;
 
-/**
- *
- */
 public interface Guardian {
-  public enum Op {
-    // these entity operations cannot be validated, they are only here 
-    // for logging purposes. a false return on validation will not be honored
+  enum Op {
+    // Connection operations. false return on validate() will result in a refusal of the network connection
+    CONNECT_SERVER,
+    CONNECT_CLIENT,
+
+    // Entity operations. Cannot be validated - false return on validate() will not be honored
     ENTITY_CREATE,
     ENTITY_FETCH,
     ENTITY_RECONFIGURE,
     ENTITY_DESTROY,
-    // false return on these operations will result in a refusal of the network connection
-    CONNECT_SERVER,
-    CONNECT_CLIENT,
-    
-    SERVER_DUMP,
-    SERVER_EXIT,
+
+    // Change categories. Cannot be validated - false return on validate() will not be honored
+    TOPOLOGY_CHANGE,
```

Comment:
Introduced meaningful categories. `SERVER_DUMP` and `SERVER_EXIT` were too specific, and `GENERIC_OP` is too vague. All the new categories warrant security auditing, and `GENERIC_OP` can be reserved for everything else.